gammapy/stats/poisson.py

```diff
@@ -413,6 +414,84 @@ def _significance_direct_on_off(n_on, n_off, alpha):
 
     return significance
 
+def Helene_ULs(excess, error, conf_level=95.45):
+    """Compute Upper limit based on Helene et al., NIM 212 (1983) 319
+
```

Comment:
We usually give a URL to the paper in the docstring.
I think it's this one?
http://adsabs.harvard.edu/abs/1984NIMPA.228..120H
Can you please add it?